tensorflow/java/src/main/java/org/tensorflow/types/TFDouble.java

```diff
@@ -0,0 +1,14 @@
+// GENERATED FILE. Edit tftypes.pl instead.
+package org.tensorflow.types;
+
+/** The class TFDouble represents 64-bit double precision floating point. */
+public class TFDouble implements Types.TFType {
+  /** Represents the type TFDouble at run time. */
+  public static final Class<TFDouble> T = TFDouble.class;
+  static {
+    Types.typeCodes.put(TFDouble.T, 2);
+  }
+  static {
+    Types.scalars.put(TFDouble.T, 0);
```

Comment:
How so? IIUC, `Types.scalars` is a `Map<Class<T>, Object>`, so where is the association between `TFDouble` and the `Object` argument to `put` being made?